submissions/description/cross_site_scripting_xss/referer/template.md

```diff
@@ -0,0 +1,30 @@
+# Referer-Based Cross-Site Scripting
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. Referer-based XSS can be found on this domain which allows an attacker to control code that is executed within a user’s browser. This occurs as the referer HTTP header is vulnerable to manipulation.
```

Comment:
A clarification on the impact here for this is worth the work, as a HTTP Referrer based XSS is not immediately exploitable unless you can manipulate the Referrer used for this.